index.html

```diff
@@ -1,26 +1,278 @@
+        <div class="row">
+          <div class="col-12">
+            <p class="text">
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim veniam, quis nostrud
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="container">
+        <div class="row">
+          <div class="col-12">
+            <button class="button header-button">hire me</button>
+          </div>
+        </div>
+      </div>
+      <div class="container">
+        <div class="row">
+          <div class="col-12">
+            <p class="darr">&darr;</p>
+          </div>
+        </div>
+      </div>
     </section>
+    <section class="about_me">
+      <div class="container">
+        <div class="row">
+          <div class="col-12">
+            <h2 class="title">About Me</h2>
+          </div>
+          <div class="col-12">
+            <p class="text">
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim veniam, quis nostrud exercitation ullamco laboris
+              nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+              reprehenderit in voluptate velit esse cillum dolore eu fugiat
+              nulla pariatur. Excepteur sint occaecat cupidatat non proident,
+              sunt in culpa qui officia
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>      
     <section class="expert-wrapper">
+      <div class="expert-wrapper__title">
+        <h1 class="expert-wrapper__title--content">Expert In</h1>
+      </div>
+      <div class="container">
+        <div class="row gx-5">
+          <div class="col-4">
+            <a class="expert-wrapper__link" href="#">
+              <img
+                class="expert-wrapper__link--icon"
+                src="img/phone_icon.svg"
+                alt=""
+            /></a>
+            <h3 class="expert-wrapper__title">Mobile Development</h3>
+            <p class="expert-wrapper__content">
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim
+            </p>
+          </div>
+          <div class="col-4">
+            <a class="expert-wrapper__link" href="#">
+              <img
+                class="expert-wrapper__link--icon"
+                src="img/desktop_icon.svg"
+                alt=""
+            /></a>
+            <h3 class="expert-wrapper__title">WEB DESIGN</h3>
+            <p class="expert-wrapper__content">
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim
+            </p>
```

Comment:
Это лучше завернуть в отдельный `<div>`, дать ему класс например ` .info-box`. Тогда можно будет раздать остальные классы примерно так:
```suggestion
            <div class="info-box expert-wrapper__info-box">
              <a class="info-box__link" href="#">
                <img
                  class="info-box__icon"
                  src="img/desktop_icon.svg"
                  alt=""
              /></a>
              <h3 class="info-box__title">WEB DESIGN</h3>
              <p class="info-box__content">
                Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
                eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
                enim ad minim
              </p>
            </div>
```
Запись `expert-wrapper__link--icon` будет значить, что это модификатор элемента `link`, что здесь не подходит